server/controllers/chartsController/helpers.js

```diff
@@ -251,41 +256,83 @@ export const postProcessData = (data, studyTotals) => {
   return processedDataBySite
 }
 
-export const mongoQueryFromFilters = (filters) => {
+export const mongoQueriesFromFilters = (filters, userAccess) => {
   if (!filters) {
     return
   }
   const activeFilters = []
-  const facet = {}
+  const facetForFilters = {}
+  const includedCriteriaFacet = {}
   const chrCritFilters = filters.chrcrit_part
     .filter((f) => f.value === TRUE_STRING)
     .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
   const includedExcludedFilters = filters.included_excluded
     .filter((f) => f.value === TRUE_STRING)
     .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
+  const sexAtBirthFilters = filters.sex_at_birth
+    .filter((f) => f.value === TRUE_STRING)
+    .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
```

Comment:
do we need to account for missing being `''`, `undefined`, or `null`?

Reply:
FWIW this ^ might be a new PR if we need to do this 😄

Reply:
During sprint planning Justin and Sylvain both mentioned that values like NaN and undefined would be an importer issue. So I feel like we don't have to account for this.

Reply:
but yes we would need a whole new pr and a strategy.